src/Language/PureScript/Docs/Types.hs

```diff
@@ -352,7 +352,7 @@ parseVersion' str =
 
 asModule :: Parse PackageError Module
 asModule =
-  Module <$> key "name" (P.moduleNameFromString <$> asString)
+  Module <$> key "name" (P.moduleNameFromString . T.pack <$> asString)
```

Comment:
I think it would probably be better to use `asText` here: http://hackage.haskell.org/package/aeson-better-errors-0.9.0.1/docs/Data-Aeson-BetterErrors.html#v:asText